pulp_2to3_migration/tests/functional/test_migration_behaviour.py

```diff
@@ -401,3 +401,95 @@ def test_no_immediate_importer(self):
 
         self.assertEqual(self.file_repo_versions_api.list(pulp3_repo.pulp_href).count, 2)
         self.assertEqual(repo_content.count, 3)
+
+    def test_importer_swap(self):
+        """
+        Test that if only importers in migration plan changed, the changes are noticed.
+
+        Importers themselves haven't changed in Pulp 2, only where they are specified in the
+        Migration Plan.
+        """
+        # run for the first time with the standard plan
+        self._load_and_run(FILE_COMPLEX_PLAN)
```

Comment:
Would importer swap impact LCEs at all, and if so, does the ISO portion of the RPM plugin utilize them in the same way as RPM?

Reply:
Swap wouldn't impact them because LCEs are handled in relation to content. The only thing that matters is that the importer is being migrated but doesn't matter which repo it relates to.
I believe that ISO LCEs and RPM LCEs are used and handled in the same way.
Great questions!